bootstrap/app.php

```diff
@@ -11,7 +11,11 @@
         health: '/up',
     )
     ->withMiddleware(function (Middleware $middleware) {
-        //
+        $middleware->alias([
+            'role' => \Spatie\Permission\Middleware\RoleMiddleware::class,
+            'permission' => \Spatie\Permission\Middleware\PermissionMiddleware::class,
+            'role_or_permission' => \Spatie\Permission\Middleware\RoleOrPermissionMiddleware::class,
```

Comment:
Clean this namespaces using code clean action command in phpstorm

Reply:
@udarakalpana has been resolved